zon_blog_extensions.php

```diff
@@ -42,8 +42,9 @@ public function __construct()
 
 		register_activation_hook( __FILE__, array( $this, 'plugin_activation' ) );
 
-		// delete formerly registered option
+		// delete formerly registered options
 		delete_option( 'zon_gdpr_activated' );
+		delete_option( 'zon_ads_deactivated' );
```

Comment:
Die Optionen tauchen ja nicht mehr auf, aber müssen trotzdem ausgeschaltet werden? Ich frag aus Neugier :-)

Reply:
Im Grunde wird das nur ein einziges Mal gebraucht.  Die gespeicherte Option könnte man auch in der DB belassen, aber ich räume sie mal lieber weg. Das braucht man nur einmalig und kann in der nächsten Plugin-Version auch gerne wieder raus… unsere DB ist halt schon so vollgemüllt aus der Zeit, als wir noch keine Activate-/Deactivate-Hooks bedient  haben, da bin ich heute eher übervorsichtig.